doc/usage/configuration.rst

```diff
@@ -3642,7 +3642,7 @@ and which failures and redirects it ignores.
 
 .. confval:: linkcheck_allowed_redirects
    :type: :code-py:`dict[str, str]`
```

Comment:
```suggestion
   :type: :code-py:`dict[str, str] | None`
```

Reply:
...except that that's not really true/valid, I suppose.  Ref: https://github.com/sphinx-doc/sphinx/blob/3c4b4e31128b5f2398fae39ab0f42af8cec16714/tests/test_builders/test_build_linkcheck.py#L764-L775